docs/integrators/snapshotless-observing-squad.md

```diff
@@ -0,0 +1,68 @@
+---
+id: snapshotless-observing-squad
+title: Snapshotless Observing Squad
+---
+
+[comment]: # (mx-abstract)
+
+This page describes the Snapshotless Observing Squad, a type of Observing Squad optimized for real-time requests such as accounts data fetching and vm-query operations.
+
+[comment]: # (mx-context-auto)
+
+## Overview
+
+Whenever a node is executing the trie snapshotting process, the accounts data fetching & vm-query operations are greatly affected.
+This is caused by the fact that the snapshotting operation consumes a large number of CPU cycles along with disk iops.
```

Comment:
I/O operations instead of `iops` (for the ones less technical). Or `IOPS` (uppercase) would also be nice.

Reply:
rephrased following @raduchis suggestion